validation-test/Evolution/test_class_add_deinit.swift

```diff
@@ -0,0 +1,37 @@
+// RUN: %target-resilience-test
```

Comment:
Do we ever test this optimized? Because that's really where I'd be concerned about this…and probably a bunch of the other ones too.

Reply:
That's a good idea. I'll have it run all test with and without -O.